ow_plexil/src/plans/ReferenceMission1.plp

```diff
@@ -97,7 +98,6 @@ ReferenceMission1: Concurrence
     else
     {
       log_error ("Failed to find ground, aborting.");
-      LibraryCall Stow();
```

Comment:
Why remove this?  Stowing the arm at completion is part of the normal conops (concept of operations).

Reply:
Since the unstow is now only occurring if IdentifySampleTarget finds a valid point, I need to move the stow call to after all arm operations are finished (Which I forgot to do before).